src/excalidraw-app/collab/Portal.tsx

```diff
@@ -158,7 +165,7 @@ class Portal {
         }
         return acc;
       },
-      [] as BroadcastedExcalidrawElement[],
+      [],
```

Comment:
this was often confusing language server after upgrading to TS 4.9. And other things do still, on folder open :(. In any case, it's always better to annotate the accumulator parameter, instead of asserting the accumulator arg.